src/client/App/index.js

```diff
@@ -15,6 +15,7 @@ import { IconUserLine, IconCheckLine } from '@instructure/ui-icons';
 import { Table } from '@instructure/ui-table';
```

Comment:
@tatikamsetti Delete everything in this file so that only the index page you created shows